R/cal-validate.R

```diff
@@ -32,28 +44,51 @@
 #'   cal_validate_logistic(Class)
 #'
 #' @inheritParams cal_estimate_logistic
-#' @param .data A `data.frame` object, or `rset` object, that contains
-#' prediction columns.
+#' @param .data An `rset` object or the results of [tune::fit_resamples()] with
+#' a `.predictions` column.
 #' @export
 cal_validate_logistic <- function(.data,
                                   truth = NULL,
                                   estimate = dplyr::starts_with(".pred_"),
-                                  smooth = TRUE,
-                                  parameters = NULL,
                                   metrics = NULL,
                                   save_details = FALSE,
                                   summarize = TRUE,
                                   ...) {
   UseMethod("cal_validate_logistic")
 }
 
+#' @export
+#' @rdname cal_validate_logistic
+cal_validate_logistic.resample_results <-
+  function(.data,
+           truth = NULL,
+           estimate = dplyr::starts_with(".pred_"),
+           metrics = NULL,
+           save_details = FALSE,
+           summarize = TRUE,
+           ...) {
+
+    truth <- tune::.get_tune_outcome_names(.data)
```

Comment:
It looks like `truth` is being set regardless if someone passes it as an command arg.  I'd suggest to either give a warning if the `truth` arg is not NULL, saying that it will be ignored, or only set `truth` to `get_tune_outcome_names()` only if the `truth` arg is not NULL. 

I have the same observation for all of the Resampled functions in this script